### ISSUE TITLE
AppVeyor build is in personal account

### ISSUE THREAD

**Issue by astrofrog** [gammapy/gammapy#818]
I noticed today that the gammapy builds are in @cdeil's AppVeyor account rather than in a gammapy organization on AppVeyor:

https://ci.appveyor.com/project/cdeil/gammapy/build/1.0.1764

It would probably be a good idea to fix this in the long run.

**Comment by cdeil** [comment]
I didn't know.
What do I have to do to change?

**Comment by astrofrog** [comment]
@cdeil - I dislike how AppVeyor handles all this compared to Travis... But to do it 'properly', I think you need to make an account for gammapy:

https://ci.appveyor.com/signup/free

And then you would need to add the build there and remove it from your account. If you want the build moved with all the history, you have to email them. Once you have the account set up, you can go to Team then add GitHub teams from gammapy so that anyone in the GitHub team can control the AppVeyor build.

Note, this came up because I was updating http://www.astropy.org/astropy-dashboard/status.html and noticed that the AppVeyor links/badges only work when the AppVeyor set-up matches the GitHub organization/user.

**Comment by astrofrog** [comment]
(A side bonus of this is that the number of builds is set per account/organization, so then gammapy would have its own queue separate from your account)

**Comment by cdeil** [comment]
I've looked at this a bit. Indeed it seems that as @astrofrog said, the change would involve creating a new account on Appveyor and then making the change. More info here:
* http://help.appveyor.com/discussions/questions/1154-appveyor-account-for-github-organizations
* https://www.appveyor.com/docs/team-setup/

I'll prefer to leave as-is for now, as there's no real benefit to making the change. Changing the milestone to "wishlist".
I have updated the Gammapy README to add a badge pointing to the Appveyor status and results just now.

@astrofrog - Is it possible to change the link in http://www.astropy.org/astropy-dashboard/status.html ?

**Comment by astrofrog** [comment]
@cdeil - the dashboard is generated automatically, but I'll see if there is a way to add an exception

**Comment by cdeil** [comment]
I'll try to resolve this for v0.10. The plan is to add a Windows build on Azure (#1886) and then to delete Appveyor for Gammapy.

**Comment by cdeil** [comment]
I have the Windows builds on Azure pipelines running in #1984 . Closing this issue now.

I deleted the Appveyor webhooks at https://github.com/gammapy/gammapy/settings/hooks

At https://ci.appveyor.com/project/cdeil/gammapy/settings I see that one can delete the project, but then all the build logs from the past years are deleted. Just leaving the settings as-is should be OK, it will keep existing build logs available, but shouldn't trigger any new builds, because Appveyor doesn't get a notification from Github anymore.
